src/main/java/com/mongodb/hibernate/jdbc/MongoPreparedStatement.java

```diff
@@ -312,4 +342,80 @@ private void checkParameterIndex(int parameterIndex) throws SQLException {
                     parameterIndex, parameterValueSetters.size()));
         }
     }
+
+    private int[] doExecuteBatch() throws SQLException {
```

Comment:
While I appreciate small methods, this one isn't pulling enough weight and should be inlined into its caller.